djpyfs/djpyfs.py

```diff
@@ -98,12 +91,19 @@ def get_osfs(namespace):
     if not os.path.exists(full_path):
         os.makedirs(full_path)
     osfs = OSFS(full_path)
-    osfs = patch_fs(osfs, namespace, lambda self, filename, timeout=0:os.path.join(djfs_settings['url_root'], namespace, filename))
+    osfs = patch_fs(osfs, namespace, lambda self, filename, timeout=0: os.path.join(djfs_settings['url_root'], namespace, filename))
     return osfs
 
 def get_s3fs(namespace):
     ''' Helper method to get_filesystem for a file system on S3 '''
-    global key_id, key_secret
+    # These imports are temporary to get to a place where we can test while not incurring a boto dependency... yet.
+    from fs.s3fs import S3FS
+    from boto.s3.connection import S3Connection
+
+    key_id = djfs_settings.get('aws_access_key_id', None)
+    key_secret = djfs_settings.get('aws_secret_access_key', None)
+    s3conn = None
```

Comment:
Probably not, looks like an artifact from when I was trying to eliminate the global. In this pass my goal was simply to do the conversion and get it testing. I have a cleanup PR coming up when this one gets in and tests clean that should fix this.